src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -340,6 +340,16 @@ public void usage(final PrintStream stream, final boolean printCommon) {
         }
     }
 
+    private String convertFromHtml(final String textToConvert) {
```

Comment:
OK, after a quick review I think I can set up a check for non-ascii in the code that builds the html docs, which will fail at build time if non-ascii is detected so whoever added the character won't be able to pass tests.